package/MDAnalysis/analysis/leaflet.py

```diff
@@ -71,14 +71,22 @@
 import warnings
 
 import numpy as np
-import networkx as NX
 
 from .. import core
 from . import distances
 from .. import selections
-
 from ..due import due, Doi
 
+
+# networkx is an optional import
+try:
+    import networkx as NX
```

Comment:
```suggestion
    import networkx as nx
```
Any reason for this to be capitalised? Networkx documentation seems to use all lowercase.

Reply:
history, other parts of the code use NX, I'm mostly doing a minimal footprint thing, imho we should just use the full name if that's what we want to do.

Reply:
Fair. I was asking mostly out of curiosity, no need to change in my view.